extensions/google-calender/README.md

```diff
@@ -0,0 +1,16 @@
+# Google Calender
+
+Search and Create sharable link to registrate!
+
+## Create Event Link Feature
+create event link to registrate you can share
```

Comment:
The wording here is a bit confusing. Do you mind rephrasing it? What do you mean to say?

Reply:
I mean to say that `Create Event Link Feature` generate an event link for anyone to subscribe to their Google Calender.
let's say we are event organizer, 
we can share event to people if we know their e-mail and subscribe the event on their Google Calender automatically.
if we don't know their e-mail we cannot do this.
this case is happen when we organize an event for unspecified large number of people.
if we generate a special link for anyone to subscribe to their Google Calender and share on websites or SNS etc, people subscribe the event on their calender so that they don't miss the event start. 

we need make a special link bymyself(it seems that there are no official tool to generate this).
so I make this feat.

Reply:
thus, I have rephrased it like below. how about it?
`Search and Create a event-link for anyone to subscribe!

## Create Event Link Feature
Create a event-link for anyone to subscribe to their Google Calender.`